evap/static/scss/components/_callouts.scss

```diff
@@ -50,6 +50,22 @@
     }
 }
 
+@mixin callout-open($extra_transition...) {
+    $transition: border-radius 1s ease, border-left-width 1s ease, padding 1s ease-in-out;
+    @if length($extra_transition) > 0 {
+        $transition: append($transition, $extra_transition);
+    }
+    transition: $transition;
+}
+
+@mixin callout-close($extra_transition...) {
+    $transition: border-radius 1s ease 1s, border-left-width 1s ease 1s, padding 1s ease-in-out;
+    @if length($extra_transition) > 0 {
+        $transition: append($transition, $extra_transition);
+    }
+    transition: $transition;
+}
+
 .callout-closable {
```

Comment:
we only ever use the closable callouts for infoboxes.
maybe we can call this `.callout-infobox`, put the min width in here, and get rid off the mixins.